assets/common.sh

```diff
@@ -0,0 +1,8 @@
+
+require_attr() {
+  local type="$1"
+  local name="$2"
+  local val=$(jq -r ".$type.$name // \"\"" < "$payload")
```

Comment:
This works fine, but just as an aside, I discovered an alternate way to interpolate vars without needing to remember to escape other characters in the filter with `--arg name value`.

e.g., `jq -r --arg type "$type" --arg name "$name"  '.$type.$name // ""'`

Reply:
Nice tip!